src/api/migrations/20190622195901-create-category.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('categories', {
+    id: {
```

Comment:
can you please use a capital letter at the beginning of the table name? i.e. : Categories

Reply:
done

Reply:
@murediane I see that you're renaming the create-token-table migration. Please, don't edit migration files like this.  Leave the token migration file exactly the way it is (check Develop branch) and create a separate migration file for categories.

Reply:
I was requested removed it from the previous request and it seems like there was a misunderstanding.  We discussed about it today  they should be there .Let me bring them back